pkg/skaffold/deploy/helm_test.go

```diff
@@ -645,6 +647,53 @@ func TestHelmDependencies(t *testing.T) {
 	}
 }
 
+func TestExpandPaths(t *testing.T) {
+	home := os.Getenv("HOME")
```

Comment:
There's a `SetEnvs` function on testutil.T that you should use to cleanup automatically.